packages/react-meteor-accounts/package.js

```diff
@@ -0,0 +1,17 @@
+/* global Package */
+
+Package.describe({
+  name: 'react-accounts',
+  summary: 'React hook for reactively tracking Meteor data',
+  version: '1.0.0',
+  documentation: 'README.md',
+  git: 'https://github.com/meteor/react-packages',
+});
+
+Package.onUse(function (api) {
+  api.versionsFrom('1.10');
```

Comment:
Let's make this from Meteor 1.12

Reply:
This was updated at some point with a range between 1.10 and 2.3. Let me know if that's okay.